test/StakingPoolPatronKYC.test.ts

```diff
@@ -453,15 +453,83 @@ describe("Staking Pool Patron KYC", function () {
       expect(snapshotRetro).to.equal(expectedSnapshot1);
     });
 
+    it("should not include a DID which stakes lower than the minimum", async () => {
```

Comment:
Just adjusting the title a bit to differentiate from the "should not create a snapshot if a staker stakes lower than the snapshot minimum" test
```suggestion
    it("should not include a staker who stakes the minimum after the snapshot", async () => {
```